### DISCUSSION TITLE
Lutris vs Steam (Proton)

### DISCUSSION THREAD

**Discussion by Joromonni** [AsherGlick/Burrito#314] [Q&A]
Im in the process of debating running Burrito and was wondering on which platform or which way of running has people had the best UX/Ease of running thus far? I know Burrito is very early in development, but I thought that getting some data on experiences wouldnt hurt.

(experiences with different DEs also appreciated. I'm on X11 KDE Plasma 6)

**Comment by jmerritt** [comment]
i got it up and running very easily in steam using this guide. https://blog.xenokore.com/installing-arcdps-and-burrito-for-guildwars-2-on-linux-steam/
